config/manifests/bases/clusterserviceversion.yaml.in

```diff
@@ -5,6 +5,33 @@ metadata:
     alm-examples: '[]'
     capabilities: Basic Install
     categories: Storage
+    containerImage: quay.io/ocs-dev/lvm-operator:latest
+    description: The ODF LVM Operator manages local storage using LVM
+    operatorframework.io/cluster-monitoring: "true"
+    operatorframework.io/initialization-resource: |-
+      {
+          "apiVersion": "lvm.topolvm.io/v1alpha1",
+          "kind": "LVMCluster",
+          "metadata": {
+            "name": "odf-lvmcluster"
+          },
+          "spec": {
+            "storage": {
+              "deviceClasses": [
+                {
+                  "name": "vg1",
+                  "thinPoolConfig": {
+                    "name": "thin-pool-1",
+                    "overprovisionRatio": 10,
+                    "sizePercent": 90
+                  }
+                }
+              ]
+            }
+          }
+        }
+    repository: https://github.com/red-hat-storage/lvm-operator
+    support: Red Hat
```

Comment:
Please remove this line as the LVMO is not officially supported yet.

Reply:
done